### PR TITLE
doc: translate zh/installation/ha-installation

### PR DESCRIPTION
**Pull request by B1F030**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
translate zh/installation/ha-installation

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+每个控制平面节点均运行有 Karmada API 服务器、Karmada 调度器及 Karmada 控制管理器的一个实例。
+Karmada API 服务器能够与多个成员集群通信，并且这些成员集群可以注册到多个 Karmada API 服务器上。
+
+每个 Karmada 控制平面节点都会创建一个本地 etcd 成员，此 etcd 成员仅与其所在节点的 Karmada API 服务器通信。
+同样的规则也适用于本地的 Karmada 控制管理器和 Karmada 调度器实例。
+
+该拓扑结构将控制平面与 etcd 成员耦合在同一节点上，相比采用外置 etcd 节点的集群，其搭建与副本管理都更为简便。
+
+然而，堆叠式集群存在耦合失效的风险。如果某节点发生故障，将同时失去一个 etcd 成员及一个 Karmada 控制平面实例，
+冗余也会受到影响。您可以通过增加更多的控制平面节点降低这一风险。
```
Comment (zhzhuang-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md:32]:
```suggestion
冗余性也会受到影响
```

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+* 堆叠控制平面节点，在此模式下，etcd 节点与控制平面节点并置在一起
+
+* 外置 etcd 节点，其中 etcd 运行在控制平面节点之外的独立节点上
+
+在搭建高可用集群前，您应当谨慎权衡每种拓扑结构的利弊。
+
+## 堆叠 etcd 拓扑结构
+
+堆叠式高可用集群是一种拓扑结构，其中 etcd 提供的分布式数据存储集群堆叠在 Karmada 控制平面节点之上。
```
Comment (zhzhuang-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md:21]:
```suggestion
堆叠式高可用集群：etcd 提供的分布式数据存储集群堆叠在 Karmada 控制平面节点之上。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+冗余也会受到影响。您可以通过增加更多的控制平面节点降低这一风险。
+
+因此，为了构建堆叠式高可用集群，您应该至少运行三个堆叠式的控制平面节点。
+
+![Karmada stacked etcd](../resources/general/karmada-stacked-etcd.png)
+
+## 外置 etcd 拓扑结构
+
+外置 etcd 高可用集群是指这样一种拓扑结构：在运行 Karmada 控制平面组件的节点组成的 Karmada 集群之外，
+单独设有 etcd 提供的分布式数据存储集群。
```
Comment (zhzhuang-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md:41]:
```suggestion
外置 etcd 高可用集群：在运行 Karmada 控制平面组件的节点组成的 Karmada 集群之外，
单独设有 etcd 提供的分布式数据存储集群。
```

**Review by zhzhuang-zju — COMMENTED** [review]
well done~ others LGTM
cc @windsonsea for another look

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+同样的规则也适用于本地的 Karmada 控制管理器和 Karmada 调度器实例。
+
+该拓扑结构将控制平面与 etcd 成员耦合在同一节点上，相比采用外置 etcd 节点的集群，其搭建与副本管理都更为简便。
+
+然而，堆叠式集群存在耦合失效的风险。如果某节点发生故障，将同时失去一个 etcd 成员及一个 Karmada 控制平面实例，
+冗余性也会受到影响。您可以通过增加更多的控制平面节点降低这一风险。
+
+因此，为了构建堆叠式高可用集群，您应该至少运行三个堆叠式的控制平面节点。
+
+![Karmada stacked etcd](../resources/general/karmada-stacked-etcd.png)
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md:36]: We can suggest the original authors to provide .svg images, if any, which would be helpful for translation.

**Review by windsonsea — COMMENTED** [review]
/lgtm

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md (hunk 4 above)
Comment (RainbowMango) [reply i18n/zh/docusaurus-plugin-content-docs/current/installation/ha-installation.md:36]: +1

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**